### ISSUE TITLE
Are we going to see pretrained W&B?

### ISSUE THREAD

**Issue by bulutthecat** [qianqianwang68/omnimotion#16]
Are there any plans on releasing the pre-trained weights and biases of this model?

**Comment by liuch37** [comment]
+1. Also is there a plan to release the inference code? Thanks.

**Comment by neuroleptics** [comment]
followed

**Comment by qianqianwang68** [comment]
Hi, 

just to clarify, this method is a test-time optimization-based method (i.e., it needs to be optimized separately on each video), so there is not a single set of pre-trained weights and biases. However, we will try to release the pretrained weights for a few sequences, to help people who want to see what the results/formats look like without having to run optimization.

As for the inference code, we are basically following the evaluation protocol of [the TAP-Vid benchmark](https://github.com/deepmind/tapnet), and we added one more metric of temporal coherence. We plan to release our inference code soon.
